web/pgadmin/browser/server_groups/servers/databases/schemas/tables/templates/tables/sql/12_plus/create.sql

```diff
@@ -38,7 +38,15 @@ CREATE {% if data.relpersistence %}UNLOGGED {% endif %}TABLE{% if add_not_exists
 
         INCLUDING STORAGE{% endif %}{% if data.like_comments %}
 
-        INCLUDING COMMENTS{% endif %}{% if data.columns|length > 0 %},
+        INCLUDING COMMENTS{% endif %}{% if data.like_compression %}
+
+        INCLUDING COMPRESSION{% endif %}{% if data.like_generated %}
```

Comment:
COMPRESSION is supported from 14 onwards. So create a new folder 14_plus and create a new file "create.sql"